src/components/AppBar/AppBar.tsx

```diff
@@ -16,7 +16,7 @@ export const useStyles = makeStyles(theme => ({
     width: "auto"
   },
   empty: {
-    flexGrow: 1
+    flexGrow: 0.435
```

Comment:
huum, a bit too pixel perfect, is there not any other means to handle it and avoid such a precision ?

Reply:
Well, I am not sure but, I think this is because of the Happyculteur logo on the left. 

If I put `0.5`, the button is centered from the logo, so it is a bit too much on the right. 
I used a grid to make it perfectly centered 😇

So, we can go for a rounded flexGrow (`0.4` or `0.5`) but it would not be centered.

Or I can remove the empty div and add style to the button, to auto margin like so:
(but samem it won't be centered either from the global page but from the logo)

```diff
-         <div className={classes.empty} />
           <Button
            variant="contained"
            color="secondary"
+           style={{margin: "0 auto"}}
            href={process.env.REACT_APP_HAPPYCULTEUR_SURVEY}
            data-e2e="SurveyButton"
          >
```
Or we have to be precise with a `0.435`, I don't know if I explain it clearly. What do you think ? Do you have any idea how we can do it ?

Reply:
You can remove the empty div as you said, purpose of it was to take the place to force the `SurveyButton` to be at the most right place. So now its useless ;)

Maybe you can try to wrap the `Button` under another div with `display:flex` & `justifyContent: center`